data/infomation_extraction_from_mail.py

```diff
@@ -0,0 +1,122 @@
+import re
+import datetime
+
+"""
+header['name'] : From -> 회사
+
+title - 메일 제목
+message - 메일 본문
+"""
+
+# 아래 4줄은 gmail API 연동되어 메일 정보 가져온 상태라고 가정
+# # 일단 집어넣자..
+# header = {}
+
+# # 메일 정보 받아온 상태라고 가정
+# header['name'] = '예시컴퍼니'
+
+# 메일 예시
+#mail = {"title": "Title of the Series: The number of the Season (Korean)", "message": "Hello Miyeon,\nI hope you are having a great day!\nWe have a new project that has become available\nif you are interested. Here’s a breakdown -\nTitle: 제목 예시\n# of episodes: (...)\nRuntime: ~45m\nClient/spec: Disney +\nContent: 작품명 예시\nGenre: Documentary\nProject Management Software: ATS\nTask: Translation\nRate: $8/hour\nTranslation Instructions: (...)\nSchedule: 8/11/2019 09:00"}
```

Comment:
https://stackoverflow.com/questions/51800165/how-to-convert-datetime-from-certain-timezone-to-utc-django-datetimefield